command/format.go

```diff
@@ -347,14 +348,14 @@ func OutputSealStatus(ui cli.Ui, client *api.Client, status *api.SealStatusRespo
 	if err != nil && strings.Contains(err.Error(), "Vault is sealed") {
 		leaderStatus = &api.LeaderResponse{HAEnabled: true}
 		err = nil
-	}
-	if err != nil {
+	} else if err != nil {
 		ui.Error(fmt.Sprintf("Error checking leader status: %s", err))
 		return 1
+	} else {
```

Comment:
I don't think this should be an else.  The change above to an `else if` is safe but gratuitous: it adds nothing, and any change in a PR that amounts to "I like it this way better" is likely to be contentious, so I would prefer you revert that one too.

Reply:
@ncabatoff thx for the review - I can revert it to retaining original (which is probably safer).

In my mind I thought it may be more contextually correct to append that when its applicable outside of any error conditions.

Reply:
@ncabatoff - I reverted back to original also adding an additional `if` in-line with above to only append HA status in the event of no errors.